test/unittests/utils/xml_parsing/test_get_parsed_resources.py

```diff
@@ -691,16 +673,32 @@ def test_get_one_absolute_iri(local_name, expected):
     assert result == expected
 
 
-def test_cleanup_simpletext():
-    original = """
+@pytest.mark.parametrize(
+    ("input_val", "expected"),
+    [
+        ('<text encoding="utf8">Text (Special &amp; Ampersand)</text>', "Text (Special & Ampersand)"),
+        (
+            """<author>
+            Cavanagh, Annie       
+        </author>""",
+            "Cavanagh, Annie",
+        ),
+        ('<text encoding="xml">text &lt;not a tag&gt; text</text>', "text <not a tag> text"),
```

Comment:
Fix the encoding, or omit it altogether.

```suggestion
        ('<text encoding="utf8">text &lt;not a tag&gt; text</text>', "text <not a tag> text"),
```